qwerty_pyrt/rust/src/wrap_ast.rs

```diff
@@ -7,7 +7,7 @@ use pyo3::{
 };
 use qwerty_ast::{ast, dbg, typecheck};
 use std::fmt;
-
+use qwerty_ast::typecheck;
```

Comment:
```suggestion
use qwerty_ast::typecheck;

```

(I imagine `cargo fmt` would add this blank line back)